tox.ini

```diff
@@ -4,18 +4,13 @@
 envlist =
     clean,
     check,
-    {py27,py33,py34,py35,py36,pypy},
```

Comment:
I decided that it is too much pain to write a package that works for all Python versions, since you need to worry about dependency matrix (not necessary)